### ISSUE TITLE
Only works with eProsima FastRTPS

### ISSUE THREAD

**Issue by RMichaelSwan** [fkie/fkie-multi-agent-suite#2]
While it's understandable from a implementation perspective why multiple DDSes are not supported right now, it would be good to document this somewhere. Specifically `mas-discovery` will not launch if you use any other DDS implementation.

**Comment by atiderko** [comment]
You are right! I have extended the readme, and when starting mas-discovery via the mas gui, the environment varibale is set  to MW_IMPLEMENTATION=rmw_fastrtps_cpp

**Comment by RMichaelSwan** [comment]
@atiderko Thank you for adding this note! Have you tested using the rest of the system with another DDS? I'm using CycloneDDS for the rest of my system right now for example.

**Comment by atiderko** [comment]
Yes, it works as long as the discovery node is started with MW_IMPLEMENTATION=rmw_fastrtps_cpp. The mas_gui does this automatically for the discovery node.

**Comment by atiderko** [comment]
It has currently been tested with rmw_cyclonedds_cpp, rmw_fastrtps_cpp, and rmw_zenoh_cpp. Therefore, I am closing this issue. If there is a need for other DDSs that this software does not work with, please open a new issue.
